src/components/EditableField/EditableField.tsx

```diff
@@ -10,7 +10,7 @@ const EditableField: React.FC<{
   amendFn: (newValue: string) => void
 }> = ({ aho, objPath, amendFn }) => {
   const result: ExceptionCode | null = isException(aho, objPath)
-  const value = get(aho, objPath, "").toString()
+  const value = (get(aho, objPath, "") ?? "").toString() // object is returned for dates
```

Comment:
because an object cannot be rendered in react we need to transform it to string but we can't use `get(aho, objPath, "")?.toString()` as that could render "undefined".